std/string.d

```diff
@@ -3177,6 +3289,89 @@ if (isConvertibleToString!Range)
     });
 }
 
+/// Ditto
+auto strip(Range, Char)(Range str, const(Char)[] chars)
```

Comment:
An additional overload would take a third argument:

```d
auto strip(Range, Char)(Range str, const(Char)[] leftChars, const(Char)[] rightChars)
```

and would strip `leftChars` on the left and `rightChars` on the right.

Reply:
Nice. I will add it

Reply:
Added the new overload as suggested and added the respective tests.